app/controllers/case_contacts_controller.rb

```diff
@@ -10,19 +10,30 @@ def index
   # GET /case_contacts/1
   # GET /case_contacts/1.json
   def show
+    @case_contact_number = CasaCase.find(@case_contact.casa_case_id).case_number
```

Comment:
In the show view, it is traditional to have something like this:
```ruby
@case_contact = CaseContact.find(params[:id])
```
From the `CaseContact`, we can then derive the `case_number` in the view (or elsewhere), which I'll indicate in a note on the view as well. 😄

Reply:
We have to find the information because the form only brings back the information in this shape:

{"id":19,
"creator_id":5,
"casa_case_id":4,
"contact_type":"youth",
"other_type_text":null,
"duration_minutes":15,
"occurred_at":"2020-04-05T00:00:00.000Z",
"created_at":"2020-04-05T23:36:26.440Z",
"updated_at":"2020-04-05T23:36:26.440Z"
}